### PR TITLE
fixes for row, col indicator updates

### PR DESCRIPTION
**Pull request by rovitotv**
This fixes issue #18 and issue #27.  The pull request updates row, col indicator if the end key is pushed or if a new line is started or on pagedown key .

### CODE REVIEW

**Comment by rovitotv** [comment]
Yes I will do that and add it to this pull request!

**Comment by rovitotv** [comment]
@scopatz I think I updated the change log correctly....I have never used rever before it looks cool!

File: news/move-cond [hunk 1]
```diff
@@ -1,14 +0,0 @@
...
-**Deprecated:**
-
-**Removed:**
-
-**Fixed:**
-
-* inconsistent handling of ctrl/meta + left and ctrl/meta + right
-  * ctrl+right used to be swapped with meta+right
-
-**Security:**
```
Comment (scopatz) [inline news/move-cond]: Please don't remove this file

**Comment by scopatz** [comment]
One minor little fix.  And thanks!

File: news/move-cond (hunk 1 above)
Comment (rovitotv) [reply news/move-cond]: Yes I can fix that.....no problem.  I thought I had to delete the file because it was in the last release.

**Comment by scopatz** [comment]
great thanks!

**Pull request merged**